config/locales/en.yml

```diff
@@ -1228,8 +1228,8 @@ en:
           deli: "Deli"
           department_store: "Department Store"
           discount: "Discount Items Shop"
-          doityourself: "Do-It-Yourself"
-          dry_cleaning: "Dry Cleaning"
+          doityourself: "Do-It-Yourself Shop"
+          dry_cleaning: "Dry Cleaner"
```

Comment:
The "Dry cleaning" term is the one exclusively used in the OSM wiki, so I think it's fine to keep the original